linux_sql/scripts/psql_docker.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
+# script usage
+#./scripts/psql_docker.sh start|stop|create [db_username][db_password]
+
+enter_cmd=$1
+db_username=$2
+db_password=$3
+
+
+#if user enter create option
+if [ "$1" == "create" ] && [ "$#" = "3" ];
+then
+	#check whether container already created
+	if [ "$(docker ps -f name= "jrvs-psql" | wc -l)" = "2" ];
+	then
+		echo "container is already created"
+       	 	exit 1
```

Comment:
fix the indentation please

Reply:
I am not sure why it shows in pull request this but in origin file and GitHub file , it didn't show indentation